camelot/handlers.py

```diff
@@ -24,19 +27,33 @@ class PDFHandler(object):
 
     Parameters
     ----------
-    filepath : str
-        Filepath or URL of the PDF file.
+    filepath : str | pathlib.Path, optional (default: None)
```

Comment:
I elected to keep the arguments separate instead of combining them like `pandas.read_csv` (or any of the others) do. Mostly to preserve the existing API kwargs